### PR TITLE
Practice3

### PR DESCRIPTION
**Pull request by AlexFlores-TrainingAssureSoft**
Factory method design pattern
Practice3

### CODE REVIEW

File: practice3/src/com/academiaDigital/softwareArchitecture/practice3/Creator/DriveCreator.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.academiaDigital.softwareArchitecture.practice3.Creator;
+
+import com.academiaDigital.softwareArchitecture.practice3.Files.IFIleOperations;
+
+public abstract class DriveCreator {
+
+    public void getUsers(String fileName){
+        System.out.println("Print all users");
```
Comment (jonathan-cs) [inline practice3/src/com/academiaDigital/softwareArchitecture/practice3/Creator/DriveCreator.java:8]: shouldn't we use here the driver? in this case IFIleOperations? the idea is being able to read from the specific file.

Comment (AlexFlores-TrainingAssureSoft) [reply practice3/src/com/academiaDigital/softwareArchitecture/practice3/Creator/DriveCreator.java]: yes, you are right
I already corrected my code

**Comment by jonathan-cs** [comment]
this looks good +1

**Review by jonathan-cs — APPROVED** [review]
this looks good +1

**Pull request merged**